### ISSUE TITLE
Warning Unresponsive script

### ISSUE THREAD

**Issue by Pablo198989** [Feodor2/Mypal#39]
Why everytime i want watch something on youtube i had this error?
To be fair only way to fix that i need close MyPal and open again.  but this is stupid because i can only watch only one video at time,  next opened tabs with youtube cause Warning Unresponsive script error.

**Comment by gdl-blue** [comment]
Yeah, happened to me too. If you don't have a proper graphics driver that supports hardware acceleration, go to about:config and set **webgl.disabled;true**. 

(But it is still slow, the JavaScript engine should be updated to a faster one)

**Comment by gdl-blue** [comment]
The PC version of YouTube is a JavaScript abomination, you can also try using the mobile version by setting the user agent override **general.useragent.override.youtube.com;Mozilla/5.0 (Linux; Android 6.0.1; RedMi Note 5 Build/RB3N5C; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/68.0.3440.91 Mobile Safari/537.36**
